airflow/hooks/base_hook.py

```diff
@@ -48,3 +48,46 @@ def get_pandas_df(self, sql):
 
     def run(self, sql):
         raise NotImplemented()
+
+
+class BaseSqlHook(BaseHook):
+    """
```

Comment:
what do you think about also putting "def insert_rows" here as well and return a notImplemented?